mathesar/tests/integration/conftest.py

```diff
@@ -67,3 +68,23 @@ def schema(create_schema, schema_name):
 @pytest.fixture
 def base_schema_url(schema, live_server):
     return f"{live_server}/{schema.database.name}/{schema.id}"
+
+
+@pytest.fixture
+def schemas_page_url(live_server, test_db_name):
+    return f"{live_server}/{test_db_name}/schemas/"
+
+
+@pytest.fixture
+def go_to_patents_data_table(page, create_table, schema_name, base_schema_url):
+    """
+    Imports the `patents.csv` data into a table named "patents" and navigates to
+    the view of that table before starting the test.
+    """
+    table_name = "patents"
+    table = create_table(table_name, schema_name)
+    table.import_verified = True
+    table.save()
+    page.goto(base_schema_url)
```

Comment:
It would be better to have the page navigation as a different fixture or as a Page object. I would prefer to keep navigation logic as part of a Page object. I have written down an incomplete example to give a rough idea.
```
class TablePage:
    def __init__(self, page, base_schema_url):
        self.page = page

    def navigate(self):
        self.get_table_entry(page, table_name).click()
```